docs/scenarios/library-authoring/fastening-libraries-to-rtm.md

```diff
@@ -12,69 +12,75 @@ ms.devlang: dotnet
 ms.assetid: 4424a947-bdf9-4775-8d48-dc350a4e0aee
 ---
 
-# How to manage package dependency versions for .NET Core RC2
+# How to manage package dependency versions for .NET Core RTM
+
+By [Phillip Carter](https://github.com/cartermp)
```

Comment:
remove this. you'll get your pretty face with the metadata. :smile: 
The only ones I've left are the co-author situation which I'm not sure how we're going to deal with those yet.
I'd also keep the file name version agnostic. Otherwise, search crawlers will have a tough time with renamings in the future.